test_util/marathon.py

```diff
@@ -11,6 +11,7 @@
 
 TEST_APP_NAME_FMT = 'integration-test-{}'
 REQUIRED_HEADERS = {'Accept': 'application/json, text/plain, */*'}
+log = logging.getLogger(__name__)
```

Comment:
This is the idiom we are moving towards in all test infrastructure and allows this VIP test in particular to dynamically control logging in an idiomatic way